### ISSUE TITLE
💅 noStaticElementInteractions false positive for event handlers on <svelte:window/>

### ISSUE THREAD

**Issue by minht11** [biomejs/biome#10636]
### Environment information

<details>

```bash

CLI:
  Version:                      2.5.0
  Color support:                true

Platform:
  CPU Architecture:             aarch64
  OS:                           macos

```

</details>

### Rule name

noStaticElementInteractions

### Playground link

https://biomejs.dev/playground/?lineWidth=120&expand=always&whitespaceSensitivity=strict&indentScriptAndStyle=true&tab=formatter&pane=Diagnostics&files.main.svelte=PABzAGMAcgBpAHAAdAAgAGwAYQBuAGcAPQAiAHQAcwAiAD4ACgAgACAAYwBvAG4AcwB0ACAAYQAgAD0AIAAnAGEAJwAKADwALwBzAGMAcgBpAHAAdAA%2BAAoACgA8AHMAdgBlAGwAdABlADoAdwBpAG4AZABvAHcACgAJAG8AbgBrAGUAeQBkAG8AdwBuAD0AewAoAGUAKQAgAD0APgAgAHsACgAJAAkAaQBmACAAKABlAC4AawBlAHkAIAA9AD0APQAgACcAIAAnACkAIAB7AAoACQAJAAkAZQAuAHAAcgBlAHYAZQBuAHQARABlAGYAYQB1AGwAdAAoACkACgAJAAkAfQAKAAkAfQB9AAoALwA%2BAAoA

### Expected result

No warning on <svelte:*> elements. https://svelte.dev/docs/svelte/svelte-window

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct
